recipes/avahi/all/conanfile.py

```diff
@@ -15,7 +19,7 @@ class AvahiConan(ConanFile):
     homepage = "https://github.com/lathiat/avahi"
     license = "LGPL-2.1-only"
     settings = "os", "arch", "compiler", "build_type"
-    generators = "pkg_config"
+    generators = "AutotoolsDeps", "PkgConfigDeps"
```

Comment:
```suggestion
```
Avoid mixing generators attribute + method, for easier review.

Reply:
Done.